File: fizzbuz.rb
n = 100

for i in (1..n).to_a do
  if i % 3 == 0
    if i % 5 == 0
      print("fizzbuz\n")
    else
      print("fizz\n")
    end
  else if i % 5 == 0
    print("buzz\n")
  else
    print(i, "\n")
  end
end
end

File: fizzbuz.rs
fn main() {
  let n = 100;

  for i in 1..(n + 1) {
    if i % 3 == 0 {
      if i % 5 == 0 {
        println!("fizzbuz");
      } else {
        println!("fizz");
      }
    } else if i % 5 == 0 {
      println!("buz");
    } else {
      println!("{}", i);
    }
  }
}